app/airq/commands/resubscribe.py

```diff
@@ -0,0 +1,21 @@
+import typing
+
+from airq.commands.base import ApiCommandHandler
+from airq.models.zipcodes import Zipcode
+
+
+class ResubscribeHandler(ApiCommandHandler):
+    def handle(self) -> typing.List[str]:
+        if self.client.zipcode is None:
+            return self._get_missing_zipcode_message()
+
+        if self.client.is_enabled_for_alerts:
+            return [
+                f"Looks like you're already watching {self.client.zipcode.zipcode}."
```

Comment:
Minor nit. Is "watching" the right word here. Maybe a marketing question. But I think it should be "Looks like you're already subscribed to <ZIPCODE> for alerts."

Reply:
Marketing question. Will defer to Lauren, but merging for now.